charts/telegraf/README.md

```diff
@@ -1,71 +1,66 @@
-# Telegraf
+# Telegraf Helm chart
 
-[Telegraf](https://github.com/influxdata/telegraf) is a plugin-driven server agent written by the folks over at [InfluxData](https://influxdata.com) for collecting & reporting metrics.
+[Telegraf](https://github.com/influxdata/telegraf) is a plugin-driven server agent used for collecting and reporting metrics.
 
-## TL;DR
+The Telegraf Helm chart uses the [Helm](https://helm.sh) package manager to bootstrap a Telegraf (`telegraf`) deployment on a [Kubernetes](http://kubernetes.io) cluster.
 
-```console
-helm repo add influxdata https://helm.influxdata.com/
-helm upgrade --install telegraf influxdata/telegraf --namespace monitoring
-```
+To see a list of available Telegraf plugins, see https://github.com/influxdata/telegraf/tree/master/plugins/.
 
-## Introduction
+## Prerequisites
 
-This chart bootstraps a `telegraf` deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+- Kubernetes 1.4+ with beta APIs enabled
```

Comment:
Is Telegraf the only Helm chart that requires Kubernetes 1.4+ with beta APIs enabled?